gamemode/modules/hungermod/sv_hungermod.lua

```diff
@@ -1,27 +1,34 @@
+
 local function HMPlayerSpawn(ply)
-	ply:setSelfDarkRPVar("Energy", 100)
+	ply:setSelfDarkRPVar("Energy", GAMEMODE.Config.maxhunger)
+	ply.LastHungerUpdate = CurTime()
 end
 hook.Add("PlayerSpawn", "HMPlayerSpawn", HMPlayerSpawn)
 
 local function HMThink()
-	if not GAMEMODE.Config.hungerspeed then return end
+	if !GAMEMODE.Config.hungerspeed then return end
+	local time = GAMEMODE.Config.hungertimer or 10
 
 	for k, v in pairs(player.GetAll()) do
-		if v:Alive() and (not v.LastHungerUpdate or CurTime() - v.LastHungerUpdate > 10) then
+		if v:Alive() and (not v.LastHungerUpdate or CurTime() - v.LastHungerUpdate > time) and !GAMEMODE.Config.hungerexcludes[v:Team()] then
 			v:hungerUpdate()
+		elseif GAMEMODE.Config.hungerexcludes[v:Team()] then
```

Comment:
I suppose this is true @FPtje although a few other communities I do dev work for also have their own select jobs that too would benefit from being excluded from hunger.

A much more global example would be Administrators on Duty. I imagine nearly every DarkRP server out there has a job for on-duty administration personnel too (I've yet to be part or even played on any that do not). They could benefit from this too.

Respect your choice either way. I'll try and sort something out if you do not wish for this in your repo.